activerecord/test/cases/adapters/sqlite3/sqlite3_adapter_test.rb

```diff
@@ -689,6 +689,61 @@ def test_strict_strings_by_default_and_false_in_database_yml
         end
       end
 
+      def test_strict_tables_is_false
+        conn = Base.sqlite3_connection(database: ":memory:", adapter: "sqlite3")
+        conn.create_table :testings do |t|
+          t.integer :dummy
+        end
+
+        assert_nothing_raised do
+          conn.exec_query("INSERT INTO testings (dummy) VALUES ('foo')")
+        end
+      end
```

Comment:
I think we should delete this test. It isn't asserting the value of anything and we already have a lot of tests testing table creation (without strict tables on) so this test is redundant.